nuxeo-importer-core/src/main/java/org/nuxeo/ecm/platform/importer/service/DefaultImporterComponent.java

```diff
@@ -92,6 +92,10 @@ public void registerContribution(Object contribution, String extensionPoint, Com
             if (descriptor.getBulkMode() != null) {
                 importerService.setBulkMode(descriptor.getBulkMode().booleanValue());
             }
+
+            if (descriptor.getEnablePerfLogging() != null) {
+                importerService.setEnablePerfLogging(descriptor.getEnablePerfLogging().booleanValue());
+            }
```

Comment:
would be nice to have a unit test checking the forward of the configuration (if checking the logs itself is too complicated and overkill)